views-pebble/src/main/java/io/micronaut/views/pebble/PebbleViewsRenderer.java

```diff
@@ -38,15 +42,18 @@
 @Requires(property = PebbleConfigurationProperties.ENABLED, notEquals = StringUtils.FALSE)
 @Requires(classes = PebbleEngine.class)
 public class PebbleViewsRenderer<T> implements ViewsRenderer<T> {
-    
+
     private final PebbleEngine engine;
+    private final HttpLocaleResolver httpLocaleResolver;
 
     /**
      * @param engine Pebble Engine
      */
     @Inject
-    public PebbleViewsRenderer(PebbleEngine engine) {    
```

Comment:
You need to create a new constructor and annotated with `@Inject` and deprecate the old constructor. 

`public PebbleViewsRenderer(PebbleEngine engine)`. Otherwise, it will be a breaking change.

Reply:
Changed it, please check. I removed the `@Inject` from the old constructor, is that correct?

Reply:
Yes, `@Inject` should only be in the new constructor. the one you want the framework to use.